### PR TITLE
Deprecate some methods in script_utils

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Deprecate methods from script_utils

# Testing this PR

Check that travis is green
Maybe other methods could be deprecated
cc @will-moore 

# Related reading

https://trello.com/c/NPvjShf8/219-review-utils-script-omeropy

### CODE REVIEW

**Comment by will-moore** [comment]
These are deprecated simply because they will be renamed?
Or moved elsewhere or removed?
If users start to see deprecation warnings, they may simply copy the methods into their own scripts.
If we're just renaming, then maybe we should put the renamed methods in place and have the deprecated methods use them, with a warning that users should use the new methods?

Some are not marked, e.g. ```createLinkFileAnnotation```, ```getObjects```.

**Comment by will-moore** [comment]
For users of the script, they'll see the warning as "stderr" but there's not much info on which method is deprecated. E.g. running thumbnail figure script
```
/opt/hudson/workspace/OMERO-DEV-merge-deploy/src/dist/lib/python/omero/util/imageUtil.py:123: DeprecationWarning: This method is deprecated as of OMERO 5.3.0
  "This method is deprecated as of OMERO 5.3.0", DeprecationWarning)
```

**Comment by jburel** [comment]
I have only marked the ones that were "legacy"
The ones currently used did not get marked. We can certainly mark them and introduce the new ones

**Comment by will-moore** [comment]
For the deprecated methods, it seems we could save a lot of code duplication if they simply delegated to use the new renamed methods.

**Comment by jburel** [comment]
@will-moore: forgot to do that. I will push changes

**Comment by jburel** [comment]
--depends-on #5139

File: components/tools/OmeroPy/src/omero/util/script_utils.py [hunk 1]
```diff
@@ -254,29 +317,43 @@ def downloadFile(rawFileStore, originalFile, filePath=None):
     @param filePath:    Where to download the file.
                         If None, use originalFile.getName().getValue()
     """
-    fileId = originalFile.getId().getValue()
-    rawFileStore.setFileId(fileId)
-    fileSize = originalFile.getSize().getValue()
-    maxBlockSize = 10000
+    warnings.warn(
+        "This method is deprecated as of OMERO 5.3.0", DeprecationWarning)
```
Comment (will-moore) [inline components/tools/OmeroPy/src/omero/util/script_utils.py]: Add "Use download_file instead" as you've done elsewhere.

**Comment by will-moore** [comment]
Tests look good at https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-integration-Python27/469/testReport/ and code changes are fine.
Just a minor comment on the warning, but otherwise good to merge.

**Comment by jburel** [comment]
@will-moore fixed

**Comment by will-moore** [comment]
Great, thanks. Good to merge.

**Pull request merged**